setup.cfg

```diff
@@ -35,6 +35,7 @@ check_untyped_defs = true
 disallow_any_generics = true
 disallow_incomplete_defs = true
 disallow_untyped_defs = true
+ignore_missing_imports = true
```

Comment:
could you share more on why we set this?

Reply:
Of course! When I run mypy on the entire repo, I get it complaining at me about three libraries missing stubs.

e.g. 

```
setup.py:4: error: No library stub file for module 'setuptools'
amundsen_common/models/user.py:7: error: Cannot find implementation or library stub for module named 'marshmallow'
amundsen_common/models/user.py:8: error: Cannot find implementation or library stub for module named 'marshmallow_annotations.ext.attrs'
```

I can set it specifically for those libraries in the cfg though, which is a less stringent option.